user/src/com/google/gwt/resources/rg/ExternalTextResourceGenerator.java

```diff
@@ -170,8 +172,8 @@ public void prepare(TreeLogger logger, ResourceContext context,
     offsets.put(method.getName(), hashes.get(toWrite));
   }
 
-  private String getMd5HashOfData() {
-    return Util.computeStrongName(Util.getBytes(data.toString()));
+  private String getMurmurHashOfData() {
```

Comment:
This method should be further renamed to be clear it is compute the hash again - or better, compute just once and memoize, as if `shouldUseJsonp()` is true we end up using it twice.